app/main.py

```diff
@@ -1,6 +1,34 @@
+import os
+
+import requests
+
+
+URL = os.environ["API_KEY"]
```

Comment:
why did u called api key as url ? use URL variable for api url and API_KEY for api key